examples/integration/pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+	- 'packages/**/*'
+	- 'demo'
```

Comment:
Is the use of pnpm here too opinionated?  Its nice being able to show how to setup a demo project, but not everyone uses pnpm

Reply:
When we had this discussion for the `component` example, we decided to remove the monorepo setup for being too opinionated and reference `npm link` from the README. I personally don't mind it here, though, especially if this leans more into being an example than a starter

Reply:
We also originally had `mocha` and `chai` in the `component` example, but decided to remove them, so I think leaving them out here is fine!